draft-ietf-httpbis-cache-latest.xml

```diff
@@ -386,6 +386,34 @@
    prohibited from storing such responses.
 </t>
 
+<section title="Storing Header and Trailer Fields" anchor="storing.fields">
+<t>
+   Caches &MUST; include all received header fields &mdash; particularly
+   unrecognised ones &mdash; when storing a response; this assures that new HTTP
+   header fields can be successfully deployed. However, the following exceptions
```

Comment:
maybe "including" instead of "particularly"?

Reply:
Done.